recipes/gapseq/build.sh

```diff
@@ -1,36 +1,20 @@
 #!/usr/bin/env bash
 
-# Installation instructions taken from https://github.com/jotech/gapseq/blob/master/docs/install.md#conda April 2024 by cmkobel
+# Installation instructions taken from https://gapseq.readthedocs.io/en/latest/install.html
 
 # Copy contents to conda prefix
-mkdir -p ${PREFIX}/gapseq/
-cp ISSUE_TEMPLATE.MD LICENSE README.md gapseq gapseq_env.yml ${PREFIX}/gapseq/
-cp -r dat/ docs/ src/ toy/ unit/ ${PREFIX}/gapseq/
+mkdir -p ${PREFIX}
+cp ISSUE_TEMPLATE.MD LICENSE README.md gapseq gapseq_env.yml ${PREFIX}
```

Comment:
Why all these files are dumped in $PREFIX ?
Usually they should be in $PREFIX/share/gapseq